src/main/java/io/github/cruciblemc/necrotempus/modules/features/glyphs/GlyphsRender.java

```diff
@@ -1,12 +1,34 @@
 package io.github.cruciblemc.necrotempus.modules.features.glyphs;
 
+import io.github.cruciblemc.necrotempus.modules.features.modernfonts.ModernFontEntry;
 import io.github.cruciblemc.necrotempus.utils.MathUtils;
 import net.minecraft.client.renderer.Tessellator;
 import net.minecraft.client.renderer.texture.TextureManager;
 import org.lwjgl.opengl.GL11;
 
 public class GlyphsRender {
 
+    public static float renderGlyph(TextureManager textureManager, ModernFontEntry entry,
+                                    float posX, float posY, boolean shadow) {
+
+        try {
+
```

Comment:
Better move the `GL11.glPushMatrix();` and `GL11.glPopMatrix();` to outside the try-catch, if an exception happens it will corrupt the matrix stack.